backend/mqttpubsub/backend.go

```diff
@@ -33,7 +36,14 @@ func NewBackend(server, username, password string) (*Backend, error) {
 	opts.SetPassword(password)
 	opts.SetOnConnectHandler(b.onConnected)
 	opts.SetConnectionLostHandler(b.onConnectionLost)
-
+	
+	if len(cafile) != 0 {
+		tlsconfig, err := NewTLSConfig(cafile)
+		if(err == nil) {
+			opts.SetClientID("ssl-client").SetTLSConfig(tlsconfig)
```

Comment:
Is there a specific reason to use the `SetClientID`? You might run into trouble when you have multiple LoRa Gateway Bridge instances. I think only one connection per client ID is allowed.